src/Font/UiOpenTypeFeatures.cc

```diff
@@ -0,0 +1,69 @@
+#include "../../ui.h"
+#include "../ui-node.h"
+#include "nbind/nbind.h"
+
+UiOpenTypeFeatures::UiOpenTypeFeatures(uiOpenTypeFeatures *feat) {
+	f = feat;
+}
+
+UiOpenTypeFeatures::UiOpenTypeFeatures() {
+	f = uiNewOpenTypeFeatures();
+}
+
+void UiOpenTypeFeatures::free() {
+	uiFreeOpenTypeFeatures(f);
+}
+
+uiOpenTypeFeatures *UiOpenTypeFeatures::getHandle() {
+	return f;
+}
+
+UiOpenTypeFeatures *UiOpenTypeFeatures::clone(UiOpenTypeFeatures *f2) {
+	return new UiOpenTypeFeatures(uiOpenTypeFeaturesClone(f2->f));
+}
+
+void UiOpenTypeFeatures::add(const char *c, uint32_t value) {
+	uiOpenTypeFeaturesAdd(f, c[0], c[1], c[2], c[3], value);
+}
+
+void UiOpenTypeFeatures::remove(const char *c) {
+	uiOpenTypeFeaturesRemove(f, c[0], c[1], c[2], c[3]);
+}
+
+// uiOpenTypeFeaturesGet() determines whether the given feature
+// tag is present in otf. If it is, *value is set to the tag's value and
+// nonzero is returned. Otherwise, zero is returned.
+//
+// Note that if uiOpenTypeFeaturesGet() returns zero, value isn't
+// changed. This is important: if a feature is not present in a
+// uiOpenTypeFeatures, the feature is NOT treated as if its
+// value was zero anyway. Script-specific font shaping rules and
+// font-specific feature settings may use a different default value
+// for a feature. You should likewise not treat a missing feature as
+// having a value of zero either. Instead, a missing feature should
+// be treated as having some unspecified default value.
+int UiOpenTypeFeatures::get(const char *c, uint32_t *value) {
+	return uiOpenTypeFeaturesGet(f, c[0], c[1], c[2], c[3], value);
+}
+
+static unsigned int UiOpenTypeFeatures__forEach(const uiOpenTypeFeatures *otf, char a, char b, char c, char d, uint32_t value, void *data) {
+	const char str[5] = {a, b, c, d, '\0'};
```

Comment:
It seems nbind (only on Windows) doesn't like the `const char[]` type too: https://ci.appveyor.com/project/parro-it/libui-node/build/1.0.195/job/l3vjnu6vvaft8048#L37

Reply:
I'm trying to convert it to a `char*`

Reply:
😠

Reply:
> 😠

Don't you like my fix or the nbind misbehaviour?

Reply:
Please don't feel insulted.
Just that Windows has a very peculiar compiler.

Reply:
> I didn't mean to insult you.

No worries, no offense. And it wasn't an insult anyway, just an emoji 😆

> 👍 works

I saw. Maybe we can open an issue on nbind repos, seems a weird hack

Reply:
> I'm more concerned about the overloaded function. Have you tried it yourself? Maybe I just missed something obvious?

No I've not. I'll do